.github/workflows/yamllint.yaml

```diff
@@ -0,0 +1,11 @@
+name: yamllint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
```

Comment:
Please add a configuration for 110 character line lengths here. See [the github action docs](https://github.com/marketplace/actions/yaml-lint) under `Config data examples:` (multi-line). You'll want a rule like this:

```
config_data: |
  extends: default
  rules:
    line-length:
      max: 110
    document-start:
      present: false
```

Reply:
Done (I hope I put it in the right place)